ValueReference/ReferenceTypesExamples/AgendaClass.cs

```diff
@@ -11,5 +12,49 @@ public class AgendaClass
         public Color AgendaColor { get; set; }
 
         public int NumberOfFiles { get; set; }
+
+        public void SetAgendaProperties(int id, string name, Color color, int numberOfFiles)
```

Comment:
This was good for the initial example, but let's delete it for now, you can create a method in agenda service that can do this